contracts/staking/MultiFeeDistribution.sol

```diff
@@ -1,12 +1,12 @@
 pragma solidity 0.7.6;
 pragma abicoder v2;
 
+import "../interfaces/IChefIncentivesController.sol";
 import "../interfaces/IMultiFeeDistribution.sol";
 import "../dependencies/openzeppelin/contracts/IERC20.sol";
 import "../dependencies/openzeppelin/contracts/SafeERC20.sol";
 import "../dependencies/openzeppelin/contracts/SafeMath.sol";
 import "../dependencies/openzeppelin/contracts/Ownable.sol";
-import "../dependencies/openzeppelin/contracts/ReentrancyGuard.sol";
```

Comment:
You remove reentrancy guards because we know and therefore trust all the reward tokens and the staking token?

Reply:
Does this have much effect on gas?

Reply:
Yes. From my tests it has an effect in the range of thousands.